src/modules/fetchMedia.js

```diff
@@ -0,0 +1,5 @@
+const fetchMedia = (url) => fetch(url)
+  .then((response) => response.json())
+  .then((response) => response.results);
+
+export default fetchMedia;
```

Comment:
Hi @KDlamini; thanks for the positive review; However, before you run the project on your local machine; Could you mind clicking on this [live demo](https://walterokumu.github.io/tv-show/dist/)? As per your local machine, the project should be running. This happens since the .env file is ignored in the .gitignore file.  Following these 1 & 2 steps will make the project run smoothly: 

Step1:

     yarn install       

Step2:

    yarn start

Reply:
Hi @AlierPM

I did run the live version as well and I got the same result.
If there are keys required, please specify that on your README.

Reply:
Hi @ KDlamini, there is a 5 second delay when fetching the data from the API, please try again as the live is working perfectly

Reply:
@WalterOkumu 

Okay, both the live and local version is working now👍🏾 

Great job!!! 🥂